bin/deploy

```diff
@@ -7,7 +7,7 @@ export JEFFREY_TOKEN=$JEFFREY_TOKEN_STAGING
 export BUCKET=$BUCKET_STAGING
 export BUCKET_PREFIX=$PREFIX_STAGING
 
-if [ $DEPLOY_ENV == "production" ];
+if [ $DEPLOY_ENV == "production" ]; then
```

Comment:
Several of the changes in this commit are in fact !fixups of previous commits :shrug: #firstworldproblems :)